live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/repository/filter/TypeFilterApplierTest.java

```diff
@@ -76,86 +73,14 @@ void mockBasicData()
     }
 
     @Test
-    void shouldNotFilterOutWhenEmptyAllowedAndEmptyDenied()
+    void shouldThrowExceptionWhenNullNodeType()
```

Comment:
All removed tests are reflected in `TypeFilterTest` class in common module. 
This sole test is sort of a smoke test that checks if `null` node type is handled correctly (exception thrown).